README.md

```diff
@@ -35,7 +35,7 @@ The Cloud Storage sink connector supports the following properties.
 
 | Name | Type|Required | Default | Description |
 |------|----------|----------|---------|-------------|
-| `provider` |String| True | null | The Cloud Storage type, such as `aws-s3`,`gcs`.|
+| `provider` |String| True | null | The Cloud Storage type, such as `aws-s3`,`google-cloud-storage`.|
```

Comment:
why not use the `gcs` for simplicity ?

Reply:
this is refers to jclouds, https://jclouds.apache.org/reference/providers/#blobstore-providers

Reply:
👍🏼